opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/RelevanceQueryBuildTest.java

```diff
@@ -79,7 +79,7 @@ void throws_SemanticCheckException_when_wrong_argument_name() {
 
     SemanticCheckException exception =
         assertThrows(SemanticCheckException.class, () -> query.build(expr));
-    assertEquals("Parameter wrongarg is invalid for mock_query function.",
+    assertEquals("Parameter wrongarg is invalid for mock_function function.",
```

Comment:
I changed this because...
1. Of a change I made in [`RelevanceQuery.java`]([.../org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/RelevanceQuery.java](https://github.com/Bit-Quill/opensearch-project-sql/pull/157/files#diff-a8e606e475c1ce9456358f4f1e598b33194d792673d3698b934f05e7ccdd6fac))
2. It is the only failing unit test with this change, and to support alternate syntaxes for functions this change seems necessary